src/foam/nanos/servlet/VirtualHostRoutingServlet.js

```diff
@@ -156,6 +156,13 @@ foam.CLASS({
             out.println("<script language=\\"javascript\\" src=\\"" + appConfig.getFoamUrl() + "\\" project=\\"" + appConfig.getPom() + "\\"></script>");
           } else {
             out.println("<script async language=\\"javascript\\" src=\\"/service/liveScriptBundler?");
+
```

Comment:
The <script> tag on the line above knows the pom from foam.poms, couldn't it just add it to the URL instead of having to expose this information through the appConfig?

Reply:
Deprecated LiveScriptBundler and remove liveScriptBundle from VirtualHostRoutingServlet https://github.com/kgrgreer/foam3/pull/2855/commits/d750b9595bdc7622bdc133f050fedc0ac846d1eb.